### ISSUE TITLE
Cannot install local library

### ISSUE THREAD

**Issue by AndrewGene** [devxoul/CocoaSeeds#63]
Seedfile...

``` ruby
local "Listen", "~/Conway\ Corp/Projects/Listen/", :files => "Listen/*.{swift,h}"
```

"Listen" (where the source files are) and "ListenCocoaSeed" are directories that are right beside each other in the Projects directory

sudo seed install gives me...

```
Anaylizing dependencies
Installing local seed: Listen
/Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/core.rb:539:in `mkdir': No such file or directory @ dir_s_mkdir - /Users/<username>/Conway\ Corp/Projects/ListenCocoaSeed/Seeds/Listen (Errno::ENOENT)
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/core.rb:539:in `install_local_seed'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/core.rb:416:in `block in install_seeds'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/core.rb:413:in `each'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/core.rb:413:in `install_seeds'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/core.rb:106:in `install'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/lib/cocoaseeds/command.rb:11:in `run'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/cocoaseeds-0.8.0/bin/seed:5:in `<top (required)>'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/bin/seed:23:in `load'
    from /Users/<username>/.rvm/rubies/ruby-2.2.2/bin/seed:23:in `<main>'
    from /Users/<username>/.rvm/gems/ruby-2.2.2@global/bin/ruby_executable_hooks:15:in `eval'
    from /Users/<username>/.rvm/gems/ruby-2.2.2@global/bin/ruby_executable_hooks:15:in `<main>'
```

Any idea what i'm doing wrong? Thanks.

**Comment by devxoul** [comment]
@AndrewGene, can you remove the backslash from the path in Seedfile and try again?

``` diff
- "~/Conway\ Corp/Projects/Listen/"
+ "~/Conway Corp/Projects/Listen/"
```

**Comment by AndrewGene** [comment]
No such luck. :( I get the same error.

**Comment by devxoul** [comment]
Hmm. This is weird. Does `/Users/<username>/Conway\ Corp/Projects/ListenCocoaSeed/Seeds` directory exist?

**Comment by AndrewGene** [comment]
Yes, it created it but then showed me the error.

**Comment by devxoul** [comment]
@AndrewGene, How about `/Users/<username>/Conway\ Corp/Projects/ListenCocoaSeed/Seeds/Listen`? Is that directory created?

PS: can you try with [0.8.1](https://github.com/devxoul/CocoaSeeds/releases/tag/0.8.1)? I'm not sure this is related but I found a bug with local seeds.

**Comment by AndrewGene** [comment]
No, "Listen" dir was not created.  I'll try 0.8.1 and let you know. Thanks.

**Comment by AndrewGene** [comment]
0.8.1 did not fix it.

**Comment by devxoul** [comment]
@AndrewGene, thanks for reporting. I could reproduce the bug and #65 is the patch for the issue. Could you please test with that?

``` console
$ git clone https://github.com/devxoul/CocoaSeeds -b escape-only-when-required
$ cd CocoaSeeds
$ make
$ cd ~/Conway\ Corp/Projects/ListenCocoaSeed
$ seed install
```

If it works, I'll merge the PR and make a new release :)

**Comment by AndrewGene** [comment]
It worked!! Thanks so much.  Very nice gem here.

**Comment by devxoul** [comment]
@AndrewGene, just released [0.8.2](https://rubygems.org/gems/cocoaseeds/versions/0.8.2) :tada: Thanks again for your bug reporting :smile:
